Clippy macOS/AgentViewController.swift

```diff
@@ -11,8 +11,7 @@ import AppKit
 class AgentViewController: NSViewController {
     var agentController: AgentController
     var agentView: AgentView
-    var isZoomed: Bool = true
-    
+    var idleTimer:Timer!
```

Comment:
`var isZoomed: Bool = true` is missing and breaks the code. Can you please add it back? :)
And I would appreciate it, if you could add a space after the `:`.
Thanks!

Reply:
Darn it! it must've been deleted when I was resolving conflict! 🤦‍♂